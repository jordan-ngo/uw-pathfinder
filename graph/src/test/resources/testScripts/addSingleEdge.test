# Create a graph A
CreateGraph A

# Add two nodes
AddNode A one
AddNode A two

# Add an edge named edgy from one to two
AddEdge A one two edgy

# Output all nodes in the graph
ListNodes A

# Output all the children of node one in graph A
ListChildren A one
