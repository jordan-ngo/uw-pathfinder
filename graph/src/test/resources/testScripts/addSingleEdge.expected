# Create a graph A
created graph A

# Add two nodes
added node one to A
added node two to A

# Add an edge named edgy from one to two
added edge edgy from one to two in A

# Output all nodes in the graph
A contains: one two

# Output all the children of node one in graph A
the children of one in A are: two(edgy)


